core/src/mindustry/entities/comp/WeaponsComp.java

```diff
@@ -81,7 +81,7 @@ void aim(float x, float y){
     }
 
     boolean canShoot(){
-        return true;
+        return reloadMultiplier > 0f;
```

Comment:
ps, this doesn't even seem to trigger at all, the method gets replaced with the `canShoot` found in `UnitComp` 😗